config/cht/tasks.js

```diff
@@ -0,0 +1,528 @@
+          //content.edd_8601 = r.fields.edd_8601;
+          //content.weeks_since_lmp = r.fields.weeks_since_lmp;
+        }
+      }
+    ],
+    events: [
+      {
+        id: "pregnancy-home-visit",
+        start: 7, //shows 1 day later - if using new Date()
+        end: 14,
+        dueDate: function (e, c, r) {
+          //console.log('dd');
+          let x = addDays(getMostRecentLMPDateForPregnancy(c, r), 12 * 7);
+          //let x = new Date('2019-06-29');
+          //console.log('due date', x);
+          return x;
+        }
+      },
+      {
+        id: "pregnancy-home-visit",
+        start: 7,
+        end: 14,
+        dueDate: function (e, c, r) {
+          //console.log('r1', r);
+          return addDays(getMostRecentLMPDateForPregnancy(c, r), 20 * 7);
+        }
+      },
+      {
+        id: "pregnancy-home-visit",
+        start: 7,
+        end: 14,
+        dueDate: function (e, c, r) {
+          //console.log('r1', r);
+          return addDays(getMostRecentLMPDateForPregnancy(c, r), 26 * 7);
+        }
+      },
+      {
+        id: "pregnancy-home-visit",
+        start: 7,
+        end: 14,
+        dueDate: function (e, c, r) {
+          //console.log('r1', r);
+          return addDays(getMostRecentLMPDateForPregnancy(c, r), 30 * 7);
+        }
+      },
+      {
+        id: "pregnancy-home-visit",
+        start: 6,
+        end: 7,
+        dueDate: function (e, c, r) {
+          //console.log('r1', r);
+          return addDays(getMostRecentLMPDateForPregnancy(c, r), 34 * 7);
+        }
+      },
+      {
+        id: "pregnancy-home-visit",
+        start: 6,
+        end: 7,
+        dueDate: function (e, c, r) {
+          //console.log('r1', r);
+          return addDays(getMostRecentLMPDateForPregnancy(c, r), 36 * 7);
+        }
+      },
+      {
+        id: "pregnancy-home-visit",
+        start: 6,
+        end: 7,
+        dueDate: function (e, c, r) {
+          //console.log('r1', r);
+          return addDays(getMostRecentLMPDateForPregnancy(c, r), 38 * 7);
+        }
+      },
+      {
+        id: "pregnancy-home-visit",
+        start: 6,
+        end: 7,
+        dueDate: function (e, c, r) {
+          //console.log('r1', r);
+          return addDays(getMostRecentLMPDateForPregnancy(c, r), 40 * 7);
+        }
+      }
+    ]
+  },
+
+  //ANC Home Visit: show every 2 weeks (Unknown LMP)
+  {
+    icon: "icon-pregnancy",
+    title: "task.anc.pregnancy_home_visit.title",
+    appliesTo: "reports",
+    appliesToType: ["pregnancy"],
+    appliesIf: function (c, r) {// If LMP date is unknown
+      const recentLMP = getMostRecentLMPDateForPregnancy(c, r);
+      const x = recentLMP === null;
+      //We only want to show until 42 weeks + 7 days
+      const y = addDays(r.reported_date, 42 * 7 + 7) >= today;
+      //console.log('applies-f', x, y, r.reported_date, today);
+      return x && y;
+    },
+
+    resolvedIf: function (c, r, event, dueDate) {
```

Comment:
This appears to be the same function as line 35? Can we encapsulate it in a reused function?